src/modules/launcher/Plugins/Microsoft.Plugin.Uri/Interfaces/IUriParser.cs

```diff
@@ -6,6 +6,6 @@ namespace Microsoft.Plugin.Uri.Interfaces
 {
     public interface IUriParser
     {
-        bool TryParse(string input, out System.Uri result, out bool isWebUri);
+        bool TryParse(string input, out System.Uri result, out bool isWebUri, out System.Uri secondResult);
```

Comment:
```suggestion
        bool TryParse(string input, out System.Uri firstResult, out bool isFirstResultWebUri, out System.Uri secondResult);
```
To make it more clear.

Reply:
Or we define first out strict as webUri and second out strict as systemUri.

Reply:
I'll try the second method, since "isWebUri" can be optimised out of the code.